.gitmodules

```diff
@@ -4,3 +4,4 @@
 [submodule "subprojects/llm-load-test"]
 	path = subprojects/llm-load-test
 	url = https://github.com/openshift-psap/llm-load-test
+	branch = main
```

Comment:
I'm not sure if this changes the fact that `subprojects/llm-load-test` is pinned to a particular commit of `openshift-psap/llm-load-test` or not?
(I'd like to keep the pinning)